functions/v2/pubsub/subscribe-to-topic/src/main/java/functions/SubscribeToTopic.java

```diff
@@ -31,13 +30,19 @@
 
   @Override
   public void accept(CloudEvent event) {
-    Gson gson = new Gson();
-    PubSubBody pubSubBody = gson.fromJson(new String(event.getData().toBytes()), PubSubBody.class);
-    String messageString =
-        new String(
-            Base64.getDecoder().decode(pubSubBody.getMessage().getData()), StandardCharsets.UTF_8);
+    logger.info("Event: " + event.getId());
+    logger.info("Event Type: " + event.getType());
 
-    logger.info(messageString);
+    if (event.getData() != null) {
```

Comment:
Would it be better to use "guards" here? i.e:

```
if (event.getData() == null) {
  return;
}

...
```

Then you don't have to wrap this in an `if` statement.

Reply:
I have no opinion here. To me, these are essentially the same

Reply:
Gotcha - I'm still in favor of the guard approach, but I'll let @lesv break the tie.

Reply:
Easier to understand inside the if, random returns, especially when wrapped (google style) are confusing.
Google doesn't allow `if (event.getData() == null) return`.  

FYI: Java 8 has the [Guards](https://docs.oracle.com/javase/8/docs/api/java/security/Guard.html) interface, but it's not applicable here.

That said, i'd really prefer `if (event.hasData()) { ... }` but that's not defined, so go with as is.